### ISSUE TITLE
IMPORT TASK: Error [ERR_STREAM_PREMATURE_CLOSE]

### ISSUE THREAD

**Issue by veglar** [ptarmiganlabs/ctrl-q#338]
### What version of Ctrl-Q are you using?

3.13.2

### What version of Node.js are you using? Not applicable if you use the standalone version of Ctrl-Q.

N/A

### What command did you use to start Ctrl-Q?

.\ctrl-q.exe task-import --auth-type cert --host my_server_name --auth-cert-file ./cert/client.pem --auth-cert-key-file ./cert/client_key.pem --auth-user-dir my_user_domain --auth-user-id vegar --file-type csv --file-name tasks.csv

### What operating system are you using?

Windows 11

### What CPU architecture are you using?

12th Gen Intel(R) Core(TM) i7-12700H

### What Qlik Sense versions are you using?

May 2023 SR7

### Describe the Bug

Inspired by the [Example: Export tasks to CSV, then import tasks from same CSV ](https://ctrl-q.ptarmiganlabs.com/docs/command/import/#example-export-tasks-to-csv-then-import-tasks-from-same-csv) I firs run an export  command of my tasks. 
```
.\ctrl-q.exe task-get ^
--auth-type cert ^
--host <my_hostname>  ^
--auth-user-dir <my_user_dir> ^
--auth-user-id veglar ^
--output-format table ^
--output-dest file ^
--output-file-name tasks.csv ^
--output-file-format csv
```

Then running this command importing the tasks.
```
.\ctrl-q.exe task-import ^
--auth-type cert ^
--host <my_hostname>  ^
--auth-user-dir <my_user_dir> ^
--auth-user-id veglie 	^
--file-type csv ^
--file-name tasks.csv
--log.evel silly
```

Then I get an error like below
```log
2023-11-06T13:30:14.360Z info: -----------------------------------------------------------
2023-11-06T13:30:14.361Z info: | Ctrl-Q
2023-11-06T13:30:14.361Z info: |
2023-11-06T13:30:14.361Z info: | Version      : 3.13.2
2023-11-06T13:30:14.361Z info: | Log level    : debug
2023-11-06T13:30:14.361Z info: |
2023-11-06T13:30:14.362Z info: | Command      : task-import
2023-11-06T13:30:14.362Z info: |              : create tasks based on definitions in a file on disk, optionally also importing apps from QVF files.
2023-11-06T13:30:14.362Z info: |
2023-11-06T13:30:14.363Z info: | Run Ctrl-Q with the '--help' option to see a list of all available options for this command.
2023-11-06T13:30:14.363Z info: |
2023-11-06T13:30:14.363Z info: | https://github.com/ptarmiganlabs/ctrl-q
2023-11-06T13:30:14.364Z info: ----------------------------------------------------------
2023-11-06T13:30:14.364Z info:
2023-11-06T13:30:14.364Z verbose: Ctrl-Q was started as a stand-alone binary: true
2023-11-06T13:30:14.364Z verbose: Ctrl-Q was started from C:/Users/veglar/Documents/Qlik/ctrl-q
2023-11-06T13:30:14.364Z verbose: Options: {
  "logLevel": "debug",
  "port": "4242",
  "schemaVersion": "12.612.0",
  "virtualProxy": "",
  "secure": true,
  "authType": "cert",
  "authCertFile": "./cert/client.pem",
  "authCertKeyFile": "./cert/client_key.pem",
  "authRootCertFile": "./cert/root.pem",
  "fileType": "csv",
  "updateMode": "create",
  "limitImportCount": 0,
  "sleepAppUpload": 1000,
  "host": "<my_hostname>",
  "authUserDir": "<my_user_dir>",
  "authUserId": "veglar",
  "fileName": "tasks.csv"
}
2023-11-06T13:30:14.364Z verbose:
2023-11-06T13:30:14.365Z verbose: About to call action handler for subcommand: task-import
2023-11-06T13:30:14.365Z debug: Auth type: cert
2023-11-06T13:30:14.365Z debug: execPath: C:/Users/veglar/Documents/Qlik/ctrl-q
2023-11-06T13:30:14.365Z debug: authCertFile: ./cert/client.pem
2023-11-06T13:30:14.365Z debug: authCertKeyFile: ./cert/client_key.pem
2023-11-06T13:30:14.365Z debug: Checking if file C:\Users\veglar\Documents\Qlik\ctrl-q\cert\client.pem exists
2023-11-06T13:30:14.366Z verbose: Certificate file C:\Users\veglar\Documents\Qlik\ctrl-q\cert\client.pem found
2023-11-06T13:30:14.366Z debug: Checking if file C:\Users\veglar\Documents\Qlik\ctrl-q\cert\client_key.pem exists
2023-11-06T13:30:14.367Z verbose: Certificate key file C:\Users\veglar\Documents\Qlik\ctrl-q\cert\client_key.pem found
2023-11-06T13:30:14.367Z verbose: Ctrl-Q was started as a stand-alone binary: true
2023-11-06T13:30:14.367Z verbose: Ctrl-Q was started from C:/Users/veglar/Documents/Qlik/ctrl-q
2023-11-06T13:30:14.367Z info: Import tasks from definitions in file "tasks.csv"
2023-11-06T13:30:14.367Z debug: Options: {
  "logLevel": "debug",
  "port": "4242",
  "schemaVersion": "12.612.0",
  "virtualProxy": "",
  "secure": true,
  "authType": "cert",
  "authCertFile": "./cert/client.pem",
  "authCertKeyFile": "./cert/client_key.pem",
  "authRootCertFile": "./cert/root.pem",
  "fileType": "csv",
  "updateMode": "create",
  "limitImportCount": 0,
  "sleepAppUpload": 1000,
  "host": "<my_hostname>",
  "authUserDir": "<my_user_dir>",
  "authUserId": "veglar",
  "fileName": "tasks.csv"
}
2023-11-06T13:30:14.367Z verbose: Getting tags from QSEoW...
2023-11-06T13:30:14.368Z debug: About to retrieve tags from QRS API.
2023-11-06T13:30:14.420Z info: Successfully retrieved 1 tags from QSEoW
2023-11-06T13:30:14.420Z verbose: Getting custom properties from QSEoW...
2023-11-06T13:30:14.437Z info: Successfully retrieved 1 custom properties from QSEoW
2023-11-06T13:30:14.437Z debug: Checking if file tasks.csv exists
2023-11-06T13:30:14.437Z verbose: Task file "tasks.csv" found
2023-11-06T13:30:14.442Z error: IMPORT TASK: Error [ERR_STREAM_PREMATURE_CLOSE]: Premature close
    at new NodeError (node:internal/errors:388:5)
    at s1.onclose (node:internal/streams/end-of-stream:142:30)
    at s1.emit (node:events:549:35)
    at emitCloseNT (node:internal/streams/destroy:132:10)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
```
Attaching my csv output:
[tasks.csv](https://github.com/ptarmiganlabs/ctrl-q/files/13267113/tasks.csv)

### Expected Behavior

I get the error described above, but I expect the task to be imported, updating my existing tasks. 

### To Reproduce

Follow my instruction found in the description section or in the documentation page: [Example: Export tasks to CSV, then import tasks from same CSV])https://ctrl-q.ptarmiganlabs.com/docs/command/import/#example-export-tasks-to-csv-then-import-tasks-from-same-csv) 

In short. 
1. Create a command that exports all tasks to a file using 'task-get`
2. Import those tasks using `task-import`

**Comment by mountaindude** [comment]
Looks like a duplicate of #323, which was fixed a week or so ago. 
Task import should be both more robust in next Ctrl-Q version, as well as support external program tasks.

**Comment by mountaindude** [comment]
Version 3.14 was just released (download [here](https://github.com/ptarmiganlabs/ctrl-q/releases/tag/v3.14.0)), it contains (among other things) changes related to this ticket.
